README.md

```diff
@@ -40,33 +39,66 @@ O O O O O
   - stepping left or right 1 field, if this field is occupied by the opponent and the occupying pawn has moved once. (en passant) The opponent's pawn is killed.
 - The game is won by the player who manages first to get a pawn to the last row on the opposite of the board.
 
-#### Strategy
+#### Strategies
```

Comment:
move strategy to top level like pawn chess. it is the main thing in the documentation